Vision/README.md

```diff
@@ -18,36 +52,81 @@ We will improve the fundamental integrity of the Web platform. The Web will not
 
 # Vision for the W3C
 
-The W3C is an association where diverse voices from around the world and industries come together to incubate and build consensus for global standards that make up the Web, serving the Vision for the Web.
-
-* We put the needs of users first: above authors, publishers, implementers, paying W3C Members, or theoretical purity.
-* This organization believes in diversity and inclusion of participants from different geographical locations, cultures, languages, accessibility needs, gender identities, and more. 
-* We strongly emphasize accessibility, internationalization, privacy, and security.
-* Our standards are rooted in a strong royalty-free patent policy and open copyright licenses, openly developed with consensus of industry and key stakeholders.
+The W3C is an association where diverse voices 
+from around the world and industries come together 
+to incubate and build consensus 
+for global standards that make up the Web, 
+serving the Vision for the Web.
+
+* We put the needs of users first: 
+above authors, publishers, implementers, 
+paying W3C Members, or theoretical purity.
+* This organization believes in diversity and inclusion 
+of participants from different geographical locations, 
+cultures, languages, accessibility needs, 
+gender identities, and more. 
+* We strongly emphasize accessibility, 
+internationalization, 
+privacy, and security.
+* Our standards are rooted in a strong royalty-free patent policy 
+and open copyright licenses, 
+openly developed with consensus of industry and key stakeholders.
 
 # Mission of the W3C
-The fundamental purpose of the W3C is to provide an open forum, where diverse voices from around the world and industries come together, incubate, and build consensus for global standards for Web technologies, including organizations of all sizes (from single-person companies to multi-nationals) and considering public feedback.
-
-As the W3C leads the Web forward, our mission is to recognize and embody fundamental values and principles into the architecture of the web.  We must become more principled in our execution of the vision of the Web.
+The fundamental purpose of the W3C is to provide an open forum, 
+where diverse voices from around the world and industries come together, 
+incubate, and build consensus for global standards for Web technologies, 
+including organizations of all sizes 
+(from single-person companies to multi-nationals) 
+and considering public feedback.
+
+As the W3C leads the Web forward, 
+our mission is to recognize and embody fundamental values and principles 
+into the architecture of the web. 
+We must become more principled in our execution of the vision of the Web.
 
 # Principles and Values
 
 We must
-* Ensure the Web is trustworthy, by ensuring security and privacy for users. 
+* Ensure the Web is trustworthy, 
+by ensuring security and privacy for users. 
```

Comment:
You need to indent this line (and all others that are continuations of a list item). Pick your indentation scheme; Tab and I of course prefer tabs ;) but you might want to go with the more conventional spaces indent.